### ISSUE TITLE
(BUG/User Error) HomeKit Not Updating

### ISSUE THREAD

**Issue by Corvetteguy19** [tonesto7/homebridge-smartthings-v2#151]
## Verify the following before opening an trouble issue

**Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!**

-  [ x] That _OAuth_ is Enabled for the SmartApp under the IDE.
-  [ x] The SmartApp and Device Handler are using the latest code available.
-  [ x] That Both the SmartApps and Device Handlers have been _Published for You_ in the IDE.

---
## About Your Setup
-  How many devices are detected?: 6
-  Mobile App Version(Not required):
-  SmartApp Version: 2.19 / 1.6.48-395
-  Device Handler Version: HOOBS 3.2.4
-  Homebridge Version: v2
-  NodeJS Version:

## Expected Behavior
When I open a door or window, HomeKit should show the door or window as open.

## Current Behavior
HomeKit displays all of my door/windows contacts as the last state they were in when restarting HOOBS.  When I open a door or window, the SmartThings app shows the door/window as being open instantly.  HomeKit doesn’t show anything opening or closing.  It’s as if homekit is not updating after the initial reload.  In addition, using the classic smartThings app I found Homebridge would be re-installed on my IDE account.  I removed everything, installed HBV2 again and this time used the new smartthings app to load it.  Everything seemed to work right this time.  There weren’t multiple installations, I confirmed the Token and APP ID matched this time, but still the same result.  Everything loads into HomeKit, but it doesn’t update it’s status.  

## Steps to Reproduce (for bugs)
**Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant**

1.
2.
3.
4.

## Context
I’m really new to all of this, and I’m sure it’s a simple setting that allows Hoobs to talk to my smartthings account.  As an FYI, I don’t have a smartThings hub.  I’m using a Konnected alarm panel which I have synced with smartthings.  I’m not sure if that is what the problem is or not.  I’m guessing not since everything is still showing up in homekit, just not updating.

Thanks

---
Please include a copy of any relevant log output to assist in tracking down the bug

**Comment by Corvetteguy19** [comment]
If you need me to pull logs, please let me know the best way to do this and I’d be happy to send them.  Thanks

**Comment by Corvetteguy19** [comment]
Any ideas on this?  I also noticed in IDE the ST-Community-Installer is green.  The HBV2 is blue.  Both are marked published.  Not sure what the different colors mean in IDE.

**Comment by Casuallynoted** [comment]
I've been having a similar issue (possibly the same issue). My devices seem to almost never update, and if they do it's very very late. I opened a door and got a notification about it almost an hour later. Sometimes I don't get notifications because it seems to just sort of skip over events like that. Smartthings is notifying me right away, though.

**Comment by Corvetteguy19** [comment]
Okay so I finally got it to work.  The devices I have connected to smartThings do not require a smartThings hub.  I was trying to get everything to work without the hub, but that wasn’t working.  So I went out and bought a SmartThings hub, got it connected, reloaded the plugin, and now everything works great for the most part.  My sensors are updating instantly and my Rachio controls seem to activate.  I am noticing my Rachio does seem to leave a zone marked as on when it finishes a run, but I think that has to do more with the Rachio part of things.  So I’m going to close this since it seems pretty clear a physical SmartThings hub was required.  It might be useful to mention that Incase someone like me is in the same boat where their devices work in SmartThings but don’t require a SmartThings hub.  Thanks
